src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -15325,6 +15325,56 @@ public static void Main()
                  );
         }
 
+        [Fact]
+        public void CS1666ERR_FixedBufferNotFixedErr()
+        {
+            var text = @"
+unsafe struct S
+{
+    public fixed int buffer[1];
+}
+
+unsafe class Test
```

Comment:
Accessing a fixed buffer in an movable object still requires `unsafe`. Could you add a test for that?

Reply:
'CS1666ERR_FixedBufferNotUnsafeErr'

---
In reply to: [172940910](https://github.com/dotnet/roslyn/pull/24966#discussion_r172940910) [](ancestors = 172940910)